dvc/repo/run.py

```diff
@@ -45,6 +45,9 @@ def run(self, fname=None, no_exec=False, single_stage=False, **kwargs):
     from dvc.stage import PipelineStage, Stage, create_stage
     from dvc.dvcfile import Dvcfile, PIPELINE_FILE
 
+    if not kwargs.get("cmd"):
+        raise InvalidArgumentError("command is not specified")
+
```

Comment:
We have a lot of tests without any `cmd`. So, a lot of tests will fail here.

Reply:
Btw, I think I know why we have a lot of tests doing that type of thing: those were mainly written before we had `dvc commit`, so that was the way we're lazy :slightly_smiling_face: